doc/releases/changelog-dev.md

```diff
@@ -111,6 +111,10 @@
 
 <h3>Bug fixes</h3>
 
+* `QNode`'s now map variations on interface names to the proper backpropagation device instead of
```

Comment:
This reads a bit confusing without prior knowledge of the exact issue.

Reply:
Hope that's clearer now.